featuretools/tests/entityset_tests/test_ww_es.py

```diff
@@ -0,0 +1,619 @@
+from datetime import datetime
+
+import dask.dataframe as dd
+import numpy as np
+import pandas as pd
+import pytest
+
+import woodwork as ww
```

Comment:
these lint fixes don't look right, what is woodwork being classified as that it gets its own block instead of being lumped in with the 3rd party libraries?

Reply:
I can revert these. I've been having some problems with linting as of yesterday, and was somewhat surprised when all of these showed up yesterday, so I'm not totally sure what's acting up.

Looking at the lint CI check, it's failing because `ERROR: Could not find a version that satisfies the requirement woodwork>=0.3.0 (from featuretools[koalas]) (from versions: 0.0.1, 0.0.2, 0.0.3, 0.0.4, 0.0.5, 0.0.6, 0.0.7, 0.0.8, 0.0.9, 0.0.10, 0.0.11, 0.1.0)`

Maybe I included Woodwork incorrectly in the requirements? At the time I started this PR, I just put the latest version of Woodwork that was out, but locally I've been using my local Woodwork repo, and Woodwork `0.3.0` is probably too low of a requirement at this point. I'm nut sure what's best here.

Reply:
You might try a fresh environment with a non-local woodwork install and see if you get the same linting suggestions.

Reply:
Still don't totally understand what state I was in yesterday that caused the linting I was seeing, but I was able to recreate it in a fresh virtual environment that had everything but Woodwork installed. Installing Woodwork fixed it.

The version thing is just that Woodwork stopped supporting python 3.6 in `0.1.0`.

Reply:
I see 3.6 support was already dropped in #1413. Does this mean we need to get all the changes in from main? Or are we okay waiting a bit on that (maybe just till this branch is merged in)

Reply:
I think we can wait until after this PR